data/json/items/generic.json

```diff
@@ -1383,6 +1383,20 @@
     "volume": "250 ml",
     "to_hit": -1
   },
+  {
+    "type": "GENERIC",
+    "id": "large_lcd_screen",
+    "symbol": ",",
+    "color": "dark_gray",
+    "name": "large LCD screen",
+    "category": "spare_parts",
+    "description": "A large backlit screen, used for displaying images.  Useful in some electronics recipes.",
+    "price": 10000,
+    "material": "plastic",
+    "weight": "3000 g",
+    "volume": "1500 ml",
```

Comment:
I don't know, isn't 1.5 liter volume too small for a "large" screen? What screen size do you have in mind?

Reply:
Yeah, this is probably far too small for a "large screen".

In order to ensure I am getting a realistic value I will base the material and weight off of [this screen](http://www.panelook.com/LD550EUE-FHB1_LG%20Display_55_LCM_parameter_24253.html).

Weight: 13 kg = 13000 g
1232mm x 708.9mm x 17.0mm=14847 mL = approx. 15000 mL

I will change the JSON to fit this data.